admin/src/pages/PluginPage/index.tsx

```diff
@@ -91,17 +108,29 @@ function PluginPage() {
   const SEE_MORE_BUTTON = useFormattedLabel('button.seeMore');
   const REFRESH_BUTTON = useFormattedLabel('button.refresh');
   const BACK_BUTTON = useFormattedLabel('button.back');
-  const CONFIRM_MSG = useFormattedLabel('confirm.message');
+
+  const [isConfirmOneDialogOpen, setIsConfirmOneDialogOpen] = useState<boolean>(false);
+  const [isConfirmAllDialogOpen, setIsConfirmAllDialogOpen] = useState<boolean>(false);
+
+  function toggleConfirmOneDialog() {
+    setIsConfirmOneDialogOpen((prev) => !prev);
+  }
+
+  function toggleConfirmAllDialog() {
+    setIsConfirmAllDialogOpen((prev) => !prev);
+  }
 
   // Callbacks
-  async function triggerGithubActions() {
-    const isConfirmed = confirm(CONFIRM_MSG);
 
-    if (!isConfirmed) return;
+  async function triggerAllGithubActions() {
+    await post(`/${pluginId}/github-actions-trigger/all`);
+    handleRefetch();
+  }
 
+  async function triggerGithubActions() {
     try {
       setLoadingTriggerButton(true);
-      await post(`/${pluginId}/github-actions-trigger`);
+      await post(`/${pluginId}/github-actions-trigger/${selectedWorkflow || '0'}`);
       setToastMsg({
```

Comment:
we have an uncaught promise here, handle error for that please

Reply:
I am not sure of which error should  I catch here because there is already toast poping up when an error occurs